src/translations/en/translations.json

```diff
@@ -227,25 +227,25 @@
-  "offer-standard-consent-copy.discount.period-6months": "After any promotional period, you will be charged {{readablePrice}} / 6 months or the then-current price, plus applicable taxes, on a recurring basis. Your subscription will automatically continue until you cancel. To cancel, log into your account, click 'Manage' next to your subscription and then click 'Unsubscribe'. By checking the box, you expressly acknowledge and agree to these terms as well as the full Terms of Service.",
-  "offer-standard-consent-copy.discount.period-month": "After any promotional period, you will be charged {{readablePrice}} / month or the then-current price, plus applicable taxes, on a recurring basis. Your subscription will automatically continue until you cancel. To cancel, log into your account, click 'Manage' next to your subscription and then click 'Unsubscribe'. By checking the box, you expressly acknowledge and agree to these terms as well as the full Terms of Service.",
-  "offer-standard-consent-copy.discount.period-season": "After any promotional period, you will be charged {{readablePrice}} / season or the then-current price, plus applicable taxes, on a recurring basis. Your subscription will automatically continue until you cancel. To cancel, log into your account, click 'Manage' next to your subscription and then click 'Unsubscribe'. By checking the box, you expressly acknowledge and agree to these terms as well as the full Terms of Service.",
-  "offer-standard-consent-copy.discount.period-week": "After any promotional period, you will be charged {{readablePrice}} / week or the then-current price, plus applicable taxes, on a recurring basis. Your subscription will automatically continue until you cancel. To cancel, log into your account, click 'Manage' next to your subscription and then click 'Unsubscribe'. By checking the box, you expressly acknowledge and agree to these terms as well as the full Terms of Service.",
-  "offer-standard-consent-copy.discount.period-year": "After any promotional period, you will be charged {{readablePrice}} / year or the then-current price, plus applicable taxes, on a recurring basis. Your subscription will automatically continue until you cancel. To cancel, log into your account, click 'Manage' next to your subscription and then click 'Unsubscribe'. By checking the box, you expressly acknowledge and agree to these terms as well as the full Terms of Service.",
+  "offer-standard-consent-copy.checkout-subscription.period-3months": "You will be charged {{readablePrice}} / 3 months or the then-current price, plus applicable taxes, on a recurring basis. Your subscription will automatically continue until you cancel. To cancel, log into your account, click 'Manage' next to your subscription and then click ‘Unsubscribe.’",
```

Comment:
All these copies were recently reviewed in widgets and updated a bit, so let's keep the same translations here, eg:
```
"offer-standard-consent-copy.checkout-subscription.period-3months": "You will be charged {{readablePrice}} / 3 months (or the then-current price), plus applicable taxes, on a recurring basis. Your subscription will automatically renew unless canceled. To cancel, log into your account, click 'Manage' next to your subscription, then select 'Unsubscribe'. By proceeding with the payment, you agree to these terms and the full Terms of Service.",
```